packages/browser-client/src/Components/ShowBlockContent.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Box,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Td,
+  Th,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+} from '@chakra-ui/react'
+import { ethers } from 'ethers'
+import { PortalNetwork } from 'portalnetwork'
+import { useEffect, useState } from 'react'
+
+interface ShowBlockContentProps {
+  portal: PortalNetwork
+}
+
+class Block {
+  parentHash: string
+  sha3Uncles: string
+  miner: string
+  stateRoot: string
+  receiptsRoot: string
+  transactionRoot: string
+  gasUsed: string
+  difficulty: string
+  number: string
+  gasLimit: string
+  transactions: string
+  timeStamp: string
+  extraData: string
+  mixHash: string
+  nonce: string
+  info: Record<string, string | number>
+  constructor(options: string[]) {
+    this.parentHash = options[0]
+    this.sha3Uncles = options[1]
+    this.miner = options[2]
+    this.stateRoot = options[3]
+    this.receiptsRoot = options[4]
+    this.transactionRoot = options[5]
+    this.gasUsed = options[6]
+    this.difficulty = options[7]
+    this.number = options[8]
+    this.gasLimit = options[9]
+    this.transactions = options[10]
+    this.timeStamp = options[11]
+    this.extraData = options[12]
+    this.mixHash = options[13]
+    this.nonce = options[14]
+    this.info = {
+      parentHash: this.parentHash,
+      sha3Uncles: this.sha3Uncles,
+      miner: this.miner,
+      stateRoot: this.stateRoot,
+      receiptsRoot: this.receiptsRoot,
+      transactionRoot: this.transactionRoot,
+      gasUsed: Number(this.gasUsed),
+      difficulty: Number(this.difficulty),
+      number: Number(this.number),
+      gasLimit: Number(this.gasLimit),
+      transactions: this.transactions,
+      timeStamp: this.timeStamp,
+      extraData: this.extraData,
+      mixHash: this.mixHash,
+      nonce: Number(this.nonce),
+    }
+  }
```

Comment:
That would actually be a great use case for our @ethereumjs/block library. 🙂 (and I guess it would also make sense, since it would make things a lot more resilient in times of value verification and different types (pre-1559, post-1559 e.g.) blocks.

No sure if you want to introduce this dependency here though.

Reply:
Good point.  We've already got the `block` dependency in the monorepo so maybe we can switch out `ethers` for `block` and make this one last dependency.

Reply:
okay thanks :)